lib/commonConfig.js

```diff
@@ -160,7 +160,7 @@ function processEnvironmentVariables() {
         config.contextBroker.ngsiVersion = process.env.IOTA_CB_NGSI_VERSION;
     }
     if (process.env.IOTA_JSON_LD_CONTEXT){
-        config.contextBroker.jsonLdContext = process.env.IOTA_JSON_LD_CONTEXT;
+        config.contextBroker.jsonLdContext = process.env.IOTA_JSON_LD_CONTEXT.split(';');
```

Comment:
Is this the usual pattern for env vars which in config.js may contain arrays? It is documented in the usage of env vars in general (or in IOTA_JSON_LD_CONTEXT env var in particular, if it isn't a general pattern)?

Reply:
I've just added a note in `doc/installationguide.md` explaining this for the IOTA_JSON_LD_CONTEXT environment variable.

I've not found a general pattern for such a use case. Some use a semicolon, others use a comma. And a few use some exotic patterns.

Reply:
Keyrock and PEP Proxy already use comma `,` for  transforming strings to arrays so this should align with them.